### PR TITLE
Few updates

### PR DESCRIPTION
**Pull request by dericed**

### CODE REVIEW

File: vrecord [hunk 1]
```diff
@@ -118,7 +118,7 @@ _maketemp(){
     else
         EXT=""
     fi
-    echo "$(mktemp -q "/tmp/$(basename "${0}").XXXXXX")${EXT}"
+    echo "$(mktemp -q -t "$(basename "${0}")")${EXT}"
```
Comment (privatezero) [inline vrecord:121]: this change is breaking on my end (it throws a 'too few X's' error). Does this work for you?

**Review by privatezero — COMMENTED** [review]
The change in `_maketemp` breaks the GUI on my end - not sure if this is a linux thing or something else

File: vrecord (hunk 1 above)
Comment (privatezero) [reply vrecord:121]: Been using that in mac when I've had the occasion - just gave it a shot in Ubuntu and it got a little weird. It's created some packaged files, but it does throw that error and it also creates an empty file when run that is called `"dvpackager: Can't create temp file, exiting....ffmetadata"`. It also appears to be trying to write that same message to chapter markers - I've attached a verbose output here. 
[dvpackager_verbose.txt](https://github.com/amiaopensource/vrecord/files/15300889/dvpackager.txt)

**Comment by dericed** [comment]
I just dropped the maketemp commit. Will get back to that later.

**Review by privatezero — APPROVED** [review]
lgtm

**Review by retokromer — APPROVED** [review]
Works on Debian 12.5.

**Pull request merged**